src/core/main.c

```diff
@@ -698,6 +698,19 @@ static int parse_config_file(void) {
 
         config_parse_many(fn, conf_dirs_nulstr, "Manager\0", config_item_table_lookup, items, false, NULL);
 
+        _cleanup_free_ char *rvalue = NULL;
+        if (get_proc_cmdline_key("systemd.default_timeout_start_sec=", &rvalue)) {
+                usec_t jobtimeout;
+                int r;
+
+                r = parse_time(rvalue, &jobtimeout, 1);
+                if (r < 0) {
+                        log_warning("Failed to set job timeout to %ss", rvalue);
+                        return 0;
+                } else
+                        arg_default_timeout_start_usec = (jobtimeout*USEC_PER_SEC);
```

Comment:
THe multiplication with USEC_PER_SEC is bogus, parse_time() already returns an "usec_t", which (as the name suggests) is a value in microseconds already.

Also, why have the "jobtimeout" variable? You can just pass a direct reference to arg_default_timeout_start_usec to parse_sec()...